cdc/model/sink.go

```diff
@@ -664,11 +664,13 @@ func (d *DDLEvent) FromRenameTablesJob(job *model.Job,
 //
 //msgp:ignore SingleTableTxn
 type SingleTableTxn struct {
-	Table     *TableName
-	TableInfo *TableInfo
-	StartTs   uint64
-	CommitTs  uint64
-	Rows      []*RowChangedEvent
+	Table            *TableName
+	TableInfo        *TableInfo
+	TableInfoVersion uint64
```

Comment:
Could you please add a comment to it to explain how we get and use it?

Reply:
Done, PTAL~